core/connection.js

```diff
@@ -245,6 +245,14 @@ Blockly.Connection.prototype.dispose = function() {
   this.dbOpposite_ = null;
 };
 
+/**
+ * @return {boolean} true if the connection is not connected or is connected to
+ *    an insertion marker, false otherwise.
+ */
+Blockly.Connection.prototype.isConnectedToNonInsertionMarker = function() {
```

Comment:
This doesn't seem to be used anywhere. Also, what about connections that would be connected except there's an insertion marker between blocks?

Reply:
Removed.